data/recommendation/style.scss

```diff
@@ -34,18 +34,16 @@ body {
     width: 100%;
   }
   .image-box {
-    width: 10%;
+    align-items: center;
+    display: flex;
```

Comment:
That's right that it's the only child. I'm simply trying to center the image within its parent div. What do you think is the best way to do this?